examples/desktop/gridplot/gridplot.py

```diff
@@ -5,12 +5,16 @@
 """
 
 # test_example = true
+# sphinx_gallery_pygfx_docs = 'screenshot'
 
 import fastplotlib as fpl
 import imageio.v3 as iio
+from wgpu.gui.auto import WgpuCanvas
 
+canvas = WgpuCanvas(size=(800, 800))
```

Comment:
As noted in `pygfx` docs, scraper looks for canvas or renderer...I think examples will need to all have `canvas` variable defined

Reply:
canvas = fig.canvas ?

Reply:
Currently, I feel like the biggest issue is that no matter how I pass the canvas, when the docs build fpl is always forcing use of a `GlfwWgpuCanvas` instead of `WgpuCanvas`

Reply:
working solution, force the canvas like so:

```
from wgpu.gui.offscreen import WgpuCanvas

canvas = WgpuCanvas(size=(800, 800))
```

Reply:
This won't be a useable example then. Check the selection logic in the gui module

Reply:
I am so silly, just need to set the `os.environ` variable before `fastplotlib` gets imported so the gui autodetects the correct thing!!

Reply:
so you can do that in the sphinx `conf.py` ?

Reply:
yes!!